RFTConfig.cpp:41:20: error: ‘invalid_argument’ is not a member of ‘std’

Continuing from #2447 and and #2449 (trying to build opm-common on Ubuntu 21.04, gcc 10.3.0) :
```
$ rm -rf build
$ git fetch
$ git merge origin/master
$ mkdir build
$ cd build
$ cmake ..
$ make
[...]
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/MSW/AICD.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/MSW/SICD.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/MSW/Valve.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/Network/Branch.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/Network/ExtNetwork.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/Network/Node.cpp.o
[ 23%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.cpp.o
[ 24%] Building CXX object CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp.o
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static std::string Opm::RFTConfig::RFT2String(Opm::RFTConfig::RFT)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:41:20: error: ‘invalid_argument’ is not a member of ‘std’
   41 |         throw std::invalid_argument("unhandled enum value");
      |                    ^~~~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static Opm::RFTConfig::RFT Opm::RFTConfig::RFTFromString(const string&)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:57:20: error: ‘invalid_argument’ is not a member of ‘std’
   57 |         throw std::invalid_argument("Unknown enum state string: " + stringValue);
      |                    ^~~~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static std::string Opm::RFTConfig::PLT2String(Opm::RFTConfig::PLT)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:71:20: error: ‘invalid_argument’ is not a member of ‘std’
   71 |         throw std::invalid_argument("unhandled enum value");
      |                    ^~~~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static Opm::RFTConfig::PLT Opm::RFTConfig::PLTFromString(const string&)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:85:20: error: ‘invalid_argument’ is not a member of ‘std’
   85 |         throw std::invalid_argument("Unknown enum state string: " + stringValue );
      |                    ^~~~~~~~~~~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static std::string Opm::RFTConfig::RFT2String(Opm::RFTConfig::RFT)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:40:5: warning: control reaches end of non-void function [-Wreturn-type]
   40 |     default:
      |     ^~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static Opm::RFTConfig::RFT Opm::RFTConfig::RFTFromString(const string&)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:58:1: warning: control reaches end of non-void function [-Wreturn-type]
   58 | }
      | ^
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static std::string Opm::RFTConfig::PLT2String(Opm::RFTConfig::PLT)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:70:5: warning: control reaches end of non-void function [-Wreturn-type]
   70 |     default:
      |     ^~~~~~~
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp: In static member function ‘static Opm::RFTConfig::PLT Opm::RFTConfig::PLTFromString(const string&)’:
/home/hakon/test/opm-common/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp:86:1: warning: control reaches end of non-void function [-Wreturn-type]
   86 | }
      | ^
make[2]: *** [CMakeFiles/opmcommon.dir/build.make:3077: CMakeFiles/opmcommon.dir/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:3339: CMakeFiles/opmcommon.dir/all] Error 2
make: *** [Makefile:160: all] Error 2
```

--- a/src/opm/io/eclipse/rst/connection.cpp
+++ b/src/opm/io/eclipse/rst/connection.cpp
@@ -17,6 +17,7 @@
 */
 
 #include <cmath>
+#include <stdexcept>
 
 #include <opm/io/eclipse/rst/header.hpp>
 #include <opm/io/eclipse/rst/connection.hpp>

--- a/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp
+++ b/src/opm/parser/eclipse/EclipseState/Schedule/RFTConfig.cpp
@@ -21,6 +21,7 @@
 
 #include <algorithm>
 #include <cassert>
+#include <stdexcept>
 #include <utility>
 
 namespace Opm {

